src/server.cpp

```diff
@@ -248,7 +248,7 @@ bool InternalServer::start() {
                             &staticHandlerCallback,
                             this,
                             MHD_OPTION_SOCK_ADDR, &sockAddr,
-                            MHD_OPTION_THREAD_POOL_SIZE, m_nbThreads,
+                            MHD_OPTION_THREAD_POOL_SIZE, &m_nbThreads,
```

Comment:
This seems to be a pretty important change.
Is it a bug on our side or an API change in libmicrohttpd ?
Anyway, the `MHD_RESULT` and `m_nbThreads` are two different changes, please move them in two different commits (with explanation)

Reply:
Tout a fait, j'ai supprimé ce passage. J'ai un doute toujours, mais je ferai une autre PR si nécessaire.

Reply:
Beside removing it from the PR, it would be nice to know if there is something wrong in our code.
Passing the value instead of a address is a real bug (if confirmed).

Reply:
I have verified, I think this is perfectly correct, like it is. Sorry.